pkg/clusterresource/alibabacloud.go

```diff
@@ -88,10 +88,6 @@ func (p *AlibabaCloudBuilder) addInstallConfigPlatform(o *Builder, ic *installer
 	}
 	ic.ControlPlane.Platform.AlibabaCloud = mppMaster
 	ic.Compute[0].Platform.AlibabaCloud = mppWorker
-
-	// Alibaba Cloud only supports manual credentials mode. Manifests including required secrets
-	// must be passed to hive via cd.spec.provisioning.manifestsConfigmapRef
-	ic.CredentialsMode = installertypes.ManualCredentialsMode
```

Comment:
we'll want something that tracks "All Alibaba clusters need to be manual", etc., without requiring folks to set the command-line flag for those platforms.

Reply:
Pushed an update that defaults manual mode for Alibaba and IBM Cloud when unset along with an `Info` log that we've defaulted it on the caller's behalf.

I don't know why this error is indented so much. :man_shrugging: 

```
./bin/hiveutil create-cluster --cloud=ibmcloud abutcher-ibmcloud
DEBU[0000] Couldn't find install logs provider environment variable. Skipping. 
INFO[0000] Defaulting credentials mode to Manual for cloud=ibmcloud  command=create-cluster
FATA[0000] Error                                         command=create-cluster error="Manual credentials mode requires --manifests containing custom Secrets with manually provisioned credentials"
```

Reply:
Well, it isn't because you're using Errorf even though there's no format :shrug:

Reply:
It's the `WithError(err)` wrapping.

```diff
-                               opt.log.WithError(err).Fatal("Error")
+                               opt.log.Fatal(err)
```

```
./bin/hiveutil create-cluster --cloud=ibmcloud abutcher-ibmcloud                                                    
DEBU[0000] Couldn't find install logs provider environment variable. Skipping. 
INFO[0000] Defaulting credentials mode to Manual for cloud=ibmcloud  command=create-cluster
FATA[0000] Manual credentials mode requires --manifests containing custom Secrets with manually provisioned credentials  command=create-cluster
```

Reply:
I'm tempted to remove the `WithError(err)` like above but probably separate to this change. Anyone have opinions about the UX of that log?

Reply:
Shrug. It's eye-catching :)